web/client/epics/maps.js

```diff
@@ -300,7 +300,7 @@ const storeDetailsInfoEpic = (action$, store) =>
                 )
                     .switchMap((attributes) => {
                         let details = find(attributes, {name: 'details'});
-                        if (!details) {
+                        if (!details || details.value === 'NODATA') {
```

Comment:
Hi @kboczkowski-clurgo
please add a unit test for the case where details are not present or the value is 'NODATA'
you can do it by cloning the "test storeDetailsInfoEpic" then using mockAxios as done in GeoStoreDAO-test.
this way you simulate the server response. 
when the stream ends with Observable.empty() you can listen for the TEST_TIMEOUT type


Please move all 'NODATA' string to a constant in epics/maps.js and the reuse it there and in tests.